variables.tf

```diff
@@ -264,6 +276,30 @@ variable "session_cloudwatch_log_group_name" {
   type        = string
 }
 
+variable "ssm_key_artifact_export_access_key_id" {
+  type        = string
+  description = "The AWS SSM Parameter Store parameter that contains the AWS access key of the user that can write to the assessment artifact export bucket (e.g. \"/assessment_artifact_export/access_key_id\")."
+  default     = "/assessment_artifact_export/access_key_id"
+}
+
+variable "ssm_key_artifact_export_bucket_name" {
+  type        = string
+  description = "The AWS SSM Parameter Store parameter that contains the name of the assessment artifact export bucket (e.g. \"/assessment_artifact_export/bucket\")."
+  default     = "/assessment_artifact_export/bucket"
+}
+
+variable "ssm_key_artifact_export_bucket_region" {
+  type        = string
+  description = "The AWS SSM Parameter Store parameter that contains the region of the assessment artifact export bucket (e.g. \"/assessment_artifact_export/region\")."
```

Comment:
I believe this region is for the IAM user that can access the bucket, as buckets themselves are considered global.

Reply:
Good catch!  I corrected this and some related items in https://github.com/cisagov/cool-assessment-terraform/pull/213/commits/cc26a3ef31d656d09c3ffbf0c845f4fe3b159c6a.